### DISCUSSION TITLE
Click attribute is needed for the image component to navigate.

### DISCUSSION THREAD

**Discussion by ThiyaDezine** [dxc-technology/halstack-react#1718] [Ideas] [answered]
Image 1
![Screenshot (596)](https://github.com/dxc-technology/halstack-react/assets/122968305/05a6ede4-7198-495d-973c-9d0d9200a1f6)

Image 2
![Screenshot (598)](https://github.com/dxc-technology/halstack-react/assets/122968305/ce00cd81-8e0d-43f1-ae49-8698357a01ee)

Clickable could go to another page or display more detailed information about the selected row data.

**Comment by GomezIvann** [comment]
Can you add more details to this discussion? I cannot fully understand your doubt from just an image...

**Reply by ThiyaDezine** [reply]
**Clickable** could go to another page or display more detailed information about the selected row data.

**Reply by jsuarezgonz** [reply]
Hi @ThiyaDezine , 
can you give us more details of the use case, why do you need the image component to be clickable?

Based on that context we have to think if it is a use case covered by another component (button for example) or if it is a need for the image component itself, so the more context you can give us the better for us to analyze it.

Thank you :)

**Reply by raquelarrojo** [reply]
@ThiyaDezine so what you mean is that in the first snapshot, the actions icons are Image components and you need to navigate through them to the second snapshot?
Is that is the case, I agree with Jairo, they should be buttons instead of images.

**Reply by ThiyaDezine** [reply]
Yes, I agree with Jairo. In this instance, "button" components are preferable over image components. Due to the button component parameter name "Text" being misleading, icons only should be used as tricky.

**Reply by kukoc** [reply]
@ThiyaDezine  should be closing this discussion then?

**Reply by ThiyaDezine** [reply, marked as answer]
Using button component am able to achive. But it has hover and down state with black icon if this approach is fine we can close the discussion.
![Screenshot (604)](https://github.com/dxc-technology/halstack-react/assets/122968305/12da8223-e301-4407-a77a-5b6933216d2a)
